lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -423,6 +423,7 @@ def _update_row(attribute_names, attempted_action = 'update')
           end
           # update 後に新しく生成したインスタンスのデータを移行する
           @_swapped_id = after_instance.swapped_id
+          self.created_at = after_instance.created_at
```

Comment:
If you want to validate yourself after `#update`, you also need to update `created_at` .

Reply:
Deleted because it is no longer needed.
Thanks!